site/content/en/_index.html

```diff
@@ -7,83 +7,99 @@
 - src: "start.jpg"
   title: "start"
 ---
-{{< blocks/cover title="Welcome to minikube" image_anchor="top" height="full" color="orange" >}}
-<div class="mx-auto">
-	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
-		Read the docs! <i class="fas fa-arrow-alt-circle-right ml-2"></i>
-	</a>
-	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/google/docsy-example">
-		Download <i class="fab fa-github ml-2 "></i>
-	</a>
-	<p class="lead mt-5">The local Kubernetes environment for macOS, Linux, and Windows.</p>
-	<div class="mx-auto mt-5">
-		{{< blocks/link-down color="info" >}}
+
+<!-- Welcome to minikube -->
+<section class="row td-box td-box--white position-relative td-box--gradient td-box--height-auto">
+	<div class="container td-arrow-down">
+		<div class="row">
+			<div class="col-sm-8">
+				<h1>Welcome to minikube!</h1>
+				<p class="lead mt-5" style="text: nowrap">Local Kubernetes, focused on application development &amp; education</p>
+				<a class="btn btn-lg btn-primary mr-3 mb-4" href="/docs/">
+					Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
+				</a>
+				<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/kubernetes/minikube">
+					View Repository <i class="fab fa-github ml-2 "></i>
+				</a>
+			</div>
+			<div class="col">
+				{{< imgproc "logo" Fit "220x220" >}}{{< /imgproc >}}
+			</div>
+		</div>
 	</div>
-</div>
-{{< /blocks/cover >}}
+</section>
 
 
-{{% blocks/lead color="primary" %}}
+<!-- Sales pitch -->
+{{< blocks/lead >}}
 
-minikube is a local Kubernetes environment focused on application developers.
+<div class="container">
+<h2>Instantly productive.</h2>
+
+A single command away from reproducing your production environment, from the comfort of localhost.
```

Comment:
I'm not 100% about "reproducing your production environment", since we don't have any multi-node support.

Would probably be more like "learning Kubernetes" or something, but I suppose for _apps_ it makes sense.